ecosystem/scenarios/test_requests.py

```
@@ -0,0 +1,212 @@
+    tmp_dir = ensure_test_directories()
+
+    # Paths
+    requests_init = Path(__file__).parent.parent / "packages" / "requests" / "src" / "requests"
+    bundled_output = tmp_dir / "requests_bundled.py"
+
+    print("🔧 Bundling requests library...")
+    result = run_cribo(
+        str(requests_init),
+        str(bundled_output),
+        emit_requirements=True,
+        tree_shake=False,  # TODO: Enable once relative import bug is fixed
```

Pylint (3.3.7)

[warning] TODO: Enable once relative import bug is fixed

(W0511)